### ISSUE TITLE
Cannot reproduce README.md graphs/results on copy task [Commit: 5c5ce66]

### ISSUE THREAD

**Issue by jagleeso** [loudinthecloud/pytorch-ntm#2]
Hello,

I'm just trying to reproduce the results/graphs [shown in the README.md](https://github.com/loudinthecloud/pytorch-ntm/blob/5c5ce66376e8032c38ef4327ca381fee145f4d0f/README.md) for the "copy" task.

I am running this on the latest master branch:
Commit [5c5ce66376e8032c38ef4327ca381fee145f4d0f](https://github.com/loudinthecloud/pytorch-ntm/commit/5c5ce66376e8032c38ef4327ca381fee145f4d0f)

## How I trained my model:
`./train.py --seed 1000 --task copy --checkpoint_interval 500 --checkpoint-path ./notebooks/copy -pbatch_size=15`

NOTE: I used a batch_size of 15 instead of the default of 1, since it seems to lead to more stable convergence rates.

I then used the [python notebook](https://github.com/loudinthecloud/pytorch-ntm/blob/5c5ce66376e8032c38ef4327ca381fee145f4d0f/notebooks/copy-task-plots.ipynb) to generate 3 graphs shown in the README.md.
For convenience when comparing, I've included both the graphs **I got**, and the graph **I expect** (taken from the README.md).

## Graph 1: Training convergence
**I got:**
![image](https://user-images.githubusercontent.com/693683/37468713-1e54d7aa-283a-11e8-9c8a-2b81dae8b244.png)
**I expect:**
![image](https://user-images.githubusercontent.com/693683/37469262-46dceda6-283b-11e8-930e-df9bd412204c.png)

## Graph 2: Training convergence (per sequence length)
**I got:**
![image](https://user-images.githubusercontent.com/693683/37468761-33f7ef20-283a-11e8-812a-b61c37d2486e.png)
**I expect:**
![image](https://user-images.githubusercontent.com/693683/37469295-56b30fa8-283b-11e8-8e8a-7f82fd67136b.png)

## Graph 3: Evaluate
**I got:**
![image](https://user-images.githubusercontent.com/693683/37468798-435cc7d8-283a-11e8-9654-c1ee13476583.png)

(The expected graph being that Outputs matches the Targets)

## Setup information
My setup is:
- OS: Ubuntu 16.04.4 LTS
- Python version: 3.6.4
- PyTorch is installed using Anaconda, `pip freeze` reports the version as: 
  `torch==0.3.0.post4`
- CUDA/cuDNN versions/libraries in use by pytorch at runtime:
```
/usr/lib/x86_64-linux-gnu/libcuda.so.384.111
/usr/local/cuda-9.0/lib64/libcublas.so.9.0.176
/usr/local/cuda-9.0/lib64/libcudart.so.9.0.176
/usr/local/cuda-9.0/lib64/libcudnn.so.7.0.5
/usr/local/cuda-9.0/lib64/libcurand.so.9.0.176
/usr/local/cuda-9.0/lib64/libcusparse.so.9.0.176
/usr/local/cuda-9.0/lib64/libnvrtc.so.9.0.176
/usr/local/cuda-9.0/lib64/libnvToolsExt.so.1.0.0
```

Let me know if there is any other information I can provide.

**Comment by loudinthecloud** [comment]
Thanks for reporting it @jagleeso, the same issue was reported to me earlier this week and the fix was to use earlier commit d8fedf63b8b07d5f5fa7e43ccef6f4fc2d78ccdc for now which seemed to work. I plan to fix it in the coming days.
